src/borg/archiver/debug_cmd.py

```diff
@@ -518,6 +574,72 @@ def build_parser_debug(self, subparsers, common_parser, mid_common_parser):
             "path", metavar="PATH", type=str, help="content for which the id-hash shall get computed"
         )
 
+        # parse_obj
+        debug_parse_obj_epilog = process_epilog(
+            """
+                This command parses the object file into metadata (as json) and uncompressed data.
+                """
+        )
+        subparser = debug_parsers.add_parser(
+            "parse-obj",
+            parents=[common_parser],
+            add_help=False,
+            description=self.do_debug_parse_obj.__doc__,
+            epilog=debug_parse_obj_epilog,
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            help="parse objectfile into meta dict and data",
+        )
+        subparser.set_defaults(func=self.do_debug_parse_obj)
+        subparser.add_argument("id", metavar="ID", type=str, help="hex object ID to get from the repo")
+        subparser.add_argument(
+            "cdata", metavar="COMPRESSED_DATA", type=str, help="path of the objectfile to parse data from"
+        )
```

Comment:
consistency: isn't this the same thing you called object_path / OBJECT_PATH below?